### ISSUE TITLE
Rename to remove the x prefix

### ISSUE THREAD

**Issue by kilasuit** [dsccommunity/DFSDsc#12]
This issue is to highlight that really this shouldn't have been merged as xDFS after the amount of confusion that these x & c prefixed modules have brought to the community.

Please consider renaming to DFS_DSC (will be filing similar issues about the other repos too)

**Comment by KarolKaczmarek** [comment]
Thanks for your input @kilasuit 

We are definitely going to be removing "x" as was communicated earlier, however the plan isn't finalized yet, so we could not publish this module without the prefix yet. 

The major issue which need to be cleared up is the versioning story, for which we have RFC pending (https://github.com/PowerShell/PowerShell-RFC/blob/master/1-Draft/RFC0004-PowerShell-Module-Versioning.md). The deadline for comments is 5/28 and shortly after that we will proceed with finishing the plan and starting the process to remove "x" from repositories. 

The bottom line is we are aware of this issue and this is high on our priority list, so should be addressed soon. I'll close this issue, as it's discussed in many different places already.
